switchmap/poller/snmp/async_snmp_manager.py

```
@@ -0,0 +1,1100 @@
+    if oid[-1] == ".":
+        return False
+
+    # Test each octet to be numeric
+    octets = oid.split(".")
+
+    # Remove the first element of the list
+    octets.pop(0)
+    for value in octets:
+        try:
+            int(value)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)